src/sentry/conf/server.py

```diff
@@ -3570,6 +3570,8 @@ def build_cdc_postgres_init_db_volume(settings: Any) -> dict[str, dict[str, str]
     "options": {"cluster": "default"},
 }
 
+ADDITIONAL_BULK_QUERY_DELETES: list[Tuple[str, str, str | None]] = []
```

Comment:
Just curious why is the last one `str | None`?

Reply:
the type of this is `(model, datetime_field, order_by)`, and the `order_by` field can be `None` so you don't order by anything